packages/browser/src/stack-parsers.ts

```diff
@@ -188,3 +188,53 @@ const extractSafariExtensionDetails = (func: string, filename: string): [string,
       ]
     : [func, filename];
 };
+
+// We should put it in the highest priority to make sure it will be executed first to handle the Next.js routes.
+const highestPriority = defaultStackLineParsers.sort((a, b) => a[0] - b[0])[0][0] - 1;
+
+/**
+ * This stack parser is used to handle the special case of Next.js routes.
+ * This case didn't be place in the default stack parser because the current regex logic is too complex to be maintained.
+ * Therefore, add this parser here as a `polyfill` is better.
+ */
+export const nextRoutesStackParser: StackLineParser = [
```

Comment:
I think we can move this into the `nextjs` package, no reason for this to be with the browser code.

Reply:
Yea. I did it before but moved it to the browser package at the end. Do you mind if there is a cycle-import in the test file? browser -> nextjs -> browser...

Reply:
moved